### DISCUSSION TITLE
Weekly Update (July 5 -> July 12)

### DISCUSSION THREAD

**Discussion by mayrinck** [mayrinck/FOSSonMicrosoftStore#25] [Announcements]
Hey there guys! 
A bunch of things happened this week, and a lot of discovery as well...

1) We finally received an answer from Microsoft regarding browser stuff... Probably there will be no way to add browsers or email clients to the store if they don't use the HTML and JS engines provided by Microsoft. This may change in Windows 11 as the posting method will also change.

2) Audacity seems to be a problem for conversion, as well as OpenShot, which means both software cannot be even started on our side, maybe just the developers or people willing to mess with the code to fix the problems. As we don't do that here, they will be skipped for now...

3) GIMP has succeeded in the store, so now we have the latest GIMP there just like on the website (F*$% GIMP Pro Scammers!), but unfortunately, as gently asked by the developer Eugen Mohr, Kdenlive will go out of the store in the future. But worry not, it will be back officially soon.

4) I've added another table to the readme file listing open-source software posted by the very own developers, this may help you guys to follow up when software gets officially supported there.

**Comment by JuliaAgostino** [comment]
Great! Thanks for the update. When you refer to Microsoft's HTML and JS engines, are you not referring to [EdgeHTML](https://en.wikipedia.org/wiki/EdgeHTML)? Maybe the new rules will only be put in place starting July 28?

I'm glad to hear that GIMP is now in the store. Maybe we should get the attention of media sources that cover Microsoft news.

**Reply by mayrinck** [reply]
> 
> 
> Great! Thanks for the update. When you refer to Microsoft's HTML and JS engines, are you not referring to [EdgeHTML](https://en.wikipedia.org/wiki/EdgeHTML)? Maybe the new rules will only be put in place starting July 28?
> 
> I'm glad to hear that GIMP is now in the store. Maybe we should get the attention of media sources that cover Microsoft news.

1) There does not specifically say about EdgeHTML, but I think it's that. I don't think Microsoft has any other HTML/JS engines actually being supported, so... Maybe they're just talking that every browser should be "Edge Classic" with another interface... 

2) For sure, but for me, I don't have time or ways to contact anyone giving news about Microsoft stuff, so... Maybe this can come later.

**Reply by JuliaAgostino** [reply]
> > Great! Thanks for the update. When you refer to Microsoft's HTML and JS engines, are you not referring to [EdgeHTML](https://en.wikipedia.org/wiki/EdgeHTML)? Maybe the new rules will only be put in place starting July 28?
> > I'm glad to hear that GIMP is now in the store. Maybe we should get the attention of media sources that cover Microsoft news.
> 
> 1. There does not specifically say about EdgeHTML, but I think it's that. I don't think Microsoft has any other HTML/JS engines actually being supported, so... Maybe they're just talking that every browser should be "Edge Classic" with another interface...
> 2. For sure, but for me, I don't have time or ways to contact anyone giving news about Microsoft stuff, so... Maybe this can come later.

Why do the Microsoft Store guidelines explicitly state that they allow Gecko and Chromium browsers then? It's all very confusing...
